homeassistant/components/weatherflow_cloud/const.py

```diff
@@ -0,0 +1,8 @@
+"""Constants for the WeatherflowCloud integration."""
+import logging
+
+DOMAIN = "weatherflow_cloud"
+_LOGGER = logging.getLogger(__package__)
```

Comment:
module wide logger shouldn't be private
```suggestion
LOGGER = logging.getLogger(__package__)
```